### PR TITLE
__WD_CapabilitiesInitialize + 'operaOptions'

### PR DESCRIPTION
**Pull request by mlipok**
### Proposed changes

added support for 'operaOptions'
### Checklist

Put an `x` in the boxes that apply. If you're unsure about any of them, don't hesitate to ask. We are here to help!<br>
This is simply a reminder of what we are going to look for before merging your code.

- [x] I have read and noticed the [CODE OF CONDUCT](https://github.com/Danp2/au3WebDriver/blob/master/docs/CODE_OF_CONDUCT.md) document
- [x] I have read and noticed the [CONTRIBUTING](https://github.com/Danp2/au3WebDriver/blob/master/docs/CONTRIBUTING.md) document
- [ ] I have added necessary documentation or screenshots (if appropriate)

### Types of changes

Please check `x` the type of change your PR introduces:

- [ ] Bugfix (change which fixes an issue)
- [x] Feature (change which adds functionality)
- [ ] Code style update (formatting, renaming)
- [ ] Refactoring (functional, structural)
- [ ] Documentation content changes
- [ ] Other (please describe)

### What is the current behavior?

lack of support for  'operaOptions'

### What is the new behavior?

added support for  'operaOptions'

### Additional context

https://github.com/Danp2/au3WebDriver/issues/187

### System under test

OPERA

### CODE REVIEW

**Comment by Danp2** [comment]
Hmmm... guessing that you didn't check out https://github.com/Danp2/au3WebDriver/commit/0de1eed9c37c93dfa22db7e0a5f066638d2be458 like I suggested previously?

You should be able to remove that entire section of code and replace with something like this --

```
Local $iIndex = _ArraySearch($_WD_SupportedBrowsers, $sBrowser, Default, Default, Default, Default, Default, $_WD_BROWSER_Name)
s_SpecificOptions_KeyName = $_WD_SupportedBrowsers[$iIndex][$_WD_BROWSER_OptionsKey]
```

**Comment by mlipok** [comment]
the reson for draft is that there is still issue with using it on
https://github.com/Danp2/au3WebDriver/pull/218

**Comment by mlipok** [comment]
but then I notice that there is a need to merge it to https://github.com/Danp2/au3WebDriver/tree/supportOpera
to have ability to test it by downloading this branch as zip and use the demo

**Comment by mlipok** [comment]
> Hmmm... guessing that you didn't check out [0de1eed](https://github.com/Danp2/au3WebDriver/commit/0de1eed9c37c93dfa22db7e0a5f066638d2be458) like I suggested previously?
> 
> You should be able to remove that entire section of code and replace with something like this --
> 
> ```
> Local $iIndex = _ArraySearch($_WD_SupportedBrowsers, $sBrowser, Default, Default, Default, Default, Default, $_WD_BROWSER_Name)
> s_SpecificOptions_KeyName = $_WD_SupportedBrowsers[$iIndex][$_WD_BROWSER_OptionsKey]
> ```

I will do this in next step.
This is even not needed to provide first working version.

**Comment by mlipok** [comment]
> Hmmm... guessing that you didn't check out [0de1eed](https://github.com/Danp2/au3WebDriver/commit/0de1eed9c37c93dfa22db7e0a5f066638d2be458) like I suggested previously?

If this is a only one problem please merge this to https://github.com/Danp2/au3WebDriver/tree/supportOpera
but then make the same comment on Draft PR here: https://github.com/Danp2/au3WebDriver/pull/213

**Comment by Danp2** [comment]
@mlipok The change should be implemented here since this PR is dealing with updates to wd_capabilities.au3. 😉

**Comment by mlipok** [comment]
> @mlipok The change should be implemented here since this PR is dealing with updates to wd_capabilities.au3. 😉

I was not intend to focus on this right now.
But Ok. I will do it like you want.

**Comment by mlipok** [comment]
@Danp2  take a look again.

**Comment by Danp2** [comment]
I see that you've identified an area (edge vs msedge) where wd_capabilities.au3 doesn't match the rest of the UDF. I hadn't noticed this before. Do you have a plan on how to correct this going forward? 

I'd rather not do the `edge ===> msedge` conversion in code if at all possible. If you think we need to maintain backwards compatibility by continuing to support `edge` in addition to `msedge`, then I think the best option is to add a duplicate entry in $_WD_SupportedBrowsers for `edge`.

**Comment by mlipok** [comment]
I have 3 reason to do make ScriptBreakingChange

1. wd_capabilities.au3 is quite new UDF
2. `Edge` is a rather rarely used browser in the context of `Au3WD UDF`
3. fix is really easy and fast to incorporate manually by coders/user in they own script

**Comment by Danp2** [comment]
That sounds good to me. Be sure to also update the function headers. You may want to change the wording so that it doesn't need updating each time a new browser is added.

**Comment by mlipok** [comment]
> Be sure to also update the function headers.

Which one?
Maybe only `_WD_UpdateDriver` should be suplemented but this is the place where always was `msedge` for this reason I changed a few month ago `wd_demo.au3`.
But when I create `wd_capabilities.au3` I was not aware about `msedge` and I used `edge` as this was previously in `wd_demo.au3`

**Comment by Danp2** [comment]
> Which one?

Any in wd_capabilities.au3 that reference `edge`

File: wd_capabilities.au3 [hunk 1]
```diff
@@ -264,7 +263,7 @@ EndFunc   ;==>_WD_CapabilitiesGet
 ;                  $s_BrowserType      - [optional] a string value. Default is ''. One of:
 ;                               | 'chrome'
 ;                               | 'firefox'
-;                               | 'edge'
+;                               | 'msedge'
```
Comment (Danp2) [inline wd_capabilities.au3:266]: This is where I was suggesting that you could simplify the entry. Either change it to something like 

`$s_BrowserType      - [optional] The browser name. Default is ''`

(without the list of valid options) or you'll have to add `Opera` to the list of valid entries.

**Review by Danp2 — APPROVED** [review]

**Pull request merged**